tests/kafkatest/tests/streams/streams_static_membership_test.py

```diff
@@ -0,0 +1,125 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.tests.test import Test
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.streams import StreamsStaticMembershipService
+from kafkatest.services.verifiable_producer import VerifiableProducer
+from kafkatest.services.zookeeper import ZookeeperService
+
+class StreamsStaticMembershipTest(Test):
+    """
+    Tests using static membership when broker points to minimum supported
+    version (2.3) or higher.
+    """
+
+    input_topic = 'inputTopic'
+    pattern = 'PROCESSED'
+
+    def __init__(self, test_context):
+        super(StreamsStaticMembershipTest, self).__init__(test_context)
+        self.topics = {
+            self.input_topic: {'partitions': 18},
```

Comment:
Since we have more stream threads on each instance, for a total of 3 * 3 = 9 threads we hope to make sure they could process some record each, to avoid the scenario where the partition gets skewed across instances.